contrib/grinder/scripts/query.py

```diff
@@ -96,13 +96,30 @@ def generate(self, time, logger):
     result = self.query_request.GET(url)
 #    logger(result.getText())
     return result
-                                                               
+
+
+class AnnotationsQuery(AbstractQuery):
+  query_interval_name = 'annotations_queries_per_interval'
+  query_name = "AnnotationsQuery"
+  test_number = 5
+
+
+  def generate(self, time, logger):
+    tenant_id = random.randint(0, default_config['num_tenants'])
+    to = time
```

Comment:
@goru97 i am using the existing num_tenants currently, but planning to have a separate num_annotations_tenants